docs/kb/api-version-3/export.md

```diff
@@ -79,6 +79,46 @@ OR
 
 __NOTE:__ The range between created_after and created_before cannot exceed 1 year. When created_before is not specified, the current date is used to gauge the interval.
 
+## Prospect
+
+### Fields
+
+Select the prospect fields that need to be exported. [Following](../../object-field-references/#prospect) are the fields that are available for prospect. The value for `fields` must be an array of strings of the available fields. 
```

Comment:
Looks like this just points to the [list of fields available from sync api](http://developer.pardot.com/kb/object-field-references/#prospect)?  I know the sync api does some joins with other tables and [not all the fields there are available in export](https://git.dev.pardot.com/Pardot/pardot/blob/master/test/integration/frontend/api/ApiExportRequestManagerIntegrationTest.php#L47-L93) - `notes` and `score` come to mind.

Reply:
Yeah I may need to follow up with PAN on what to do here.

Reply:
@jpearson-sfdc  As nick noted above the fields for the sync api do not match up. Do you have a preference on how we should handle this?

Reply:
Should we denote using a column on the object field references page?